docs/io/unified.rst

```diff
@@ -352,23 +359,24 @@ the final conversion from ``(jd1, jd2)`` requires a software implementation whic
 fully compliant with the FITS Time standard.
 
 Taking this into consideration, the functionality to read/write Time
-from/to FITS is off by default. Users can store the time
+from/to FITS can be explicitly chosen to be off. Users can store the time
 representation values in the format specified by the ``format`` attribute
 of the `~astropy.time.Time` column, instead of the ``(jd1, jd2)`` format, with
-no extra metadata in the header. This is the "lossy" version,
-but meets common-use needs. For the above example, the FITS column corresponding
+no extra metadata in the header. This is the "lossy" version, but meets
+common-use needs. For the above example, the FITS column corresponding
 to ``t['a']`` will then store ``[100.0 200.0]`` instead of
-``[[ 2400100.5, 0. ], [ 2400200.5, 0. ]]``.
-This will be useful when dealing with other software packages which do not implement
-the full FITS time standard or when users want to serialize their Time object
-in the format of their choosing.
+``[[ 2400100.5, 0. ], [ 2400200.5, 0. ]]``, and this is done by setting
```

Comment:
I think I would just write ``. This is done by using a special ``info.serialze_method`` attribute, as in the following example::`` (@taldcroft - I was going to suggest to put a link to the `ecsv` documentation about the similar use there, but couldn't find it - did we actually document this?)